shard.yml

```diff
@@ -0,0 +1,8 @@
+name:
+version: 0.1.0
+author: Luis Ferreira
+
+dependencies:
+  kemal:
+    github: sdogruyol/kemal
+    branch: master
```

Comment:
Isn't there a specific version we can use, instead of edge? I'm guessing the `shard.lock` will kind of prevent this, but it would be clearer, I think.